Allow restricting cultures creation with any arbitrary names in Globalization Invariant Mode (#54247)
dotnet/runtime: docs/workflow/trimming/feature-switches.md

| EnableUnsafeBinaryFormatterSerialization | System.Runtime.Serialization.EnableUnsafeBinaryFormatterSerialization | BinaryFormatter serialization support is trimmed when set to false |
| EventSourceSupport | System.Diagnostics.Tracing.EventSource.IsSupported | Any EventSource related code or logic is trimmed when set to false |
| InvariantGlobalization | System.Globalization.Invariant | All globalization specific code and data is trimmed when set to true |

> | PredefinedCulturesOnly | System.Globalization.PredefinedCulturesOnly |  Don't allow creating a made-up cultures when the platform does not carry data for it |

Reviewer: `a made-up cultures` isn't grammatically correct.

How about 

```suggestion
| PredefinedCulturesOnly | System.Globalization.PredefinedCulturesOnly |  Don't allow creating a culture for which the platform does not have data |
```

?

Author: > Don't allow creating a culture for which the platform does not have data

feeling something missing. may be adding word `for` at the end make it more understandable.
